tests/Transformers/RegexFilterTest.php

```diff
@@ -32,6 +32,7 @@ protected function setUp() : void
             ['I was not proud of what I had learned, but I never doubted that it was worth $$$ knowing..'],
             ['Too weird to live, support@rubixml.com too rare to die https://rubixml.com'],
             ['A man who procrastinates in @his choosing will inevitably have his choice made for him by #circumstance'],
+            ['The quick quick brown fox jumped over the lazy man sitting at a bus stop.'],
```

Comment:
Can we also test words in different case? Ex. "Yo yo what's up?"

Reply:
Ahh yeah, sure. As the regex includes the case insensitive (`i`) unicode (`u`) flags i'll ensure test cases for both these variants too.